src/create_disk.sh

```diff
@@ -85,6 +85,9 @@ else
 	ext="AA64"
 fi
 
+# we use pure BLS, so don't need grub2-mkconfig
+ostree config --repo rootfs/ostree/repo set sysroot.bootloader none
```

Comment:
Ah, yes.  Cool!  This is a concrete improvement from the "drop anaconda" work, will also remove my least-favorite thing: `os-prober`.

Reply:
Yeah, I was really pleased to see it have a net positive effect on finalization time too.